app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    output_list = []
+
+    for person in people:
+        pr = Person(name=person["name"], age=person["age"])
```

Comment:
`pr` is not an informative name of variable

It's better to write like this
`output_list.append(Person(name=person["name"], age=person["age"]))`